vendor-extra/JatsContentBundle/tests/EventListener/BuildView/GraphicImageListenerTest.php

```diff
@@ -0,0 +1,268 @@
+<?php
+
+declare(strict_types=1);
+
+namespace tests\Libero\JatsContentBundle\EventListener\BuildView;
+
+use Libero\JatsContentBundle\EventListener\BuildView\GraphicImageListener;
+use Libero\ViewsBundle\Event\BuildViewEvent;
+use Libero\ViewsBundle\Views\TemplateView;
+use PHPUnit\Framework\TestCase;
+use tests\Libero\LiberoPageBundle\XmlTestCase;
+
+final class GraphicImageListenerTest extends TestCase
+{
+    use XmlTestCase;
+
+    /**
+     * @test
+     * @dataProvider nodeProvider
+     */
+    public function it_does_nothing_if_it_is_not_a_jats_graphic_element(string $xml) : void
+    {
+        $listener = new GraphicImageListener();
+
+        $element = $this->loadElement($xml);
+
+        $event = new BuildViewEvent($element, new TemplateView(null));
+        $listener->onBuildView($event);
+        $view = $event->getView();
+
+        $this->assertInstanceOf(TemplateView::class, $view);
+        $this->assertNull($view->getTemplate());
+        $this->assertEmpty($view->getArguments());
+        $this->assertEmpty($view->getContext());
+    }
+
+    public function nodeProvider() : iterable
+    {
+        yield 'different namespace' => [
+            <<< XML
+<graphic xmlns="http://example.com" xmlns:xlink="http://www.w3.org/1999/xlink"
+    xlink:href="http://example.com/image.jpg"/>
+XML
+            ,
```

Comment:
random comma mandated by coding standards?

Reply:
PHP 7.3 would make it better (https://wiki.php.net/rfc/flexible_heredoc_nowdoc_syntaxes).

Bit ugly, but 🤷‍♂️.